DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/sample.json

```diff
@@ -0,0 +1,85 @@
+{
+  "guid": "418AA4D8-0A61-4FDB-845A-CDA4DA0655E7",
```

Comment:
You need to generate a new guid, check the "guid" section on this page: https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification

Reply:
already did